spec/restaurant_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'restaurant'
+
+describe Restaurant do
+  let(:restaurant) { described_class.new('Beef & Beer', 'Takeaway') }
+
+  it 'has a menu upon initialization' do
+    expect(restaurant.menu).to be_an_instance_of Menu
+  end
+
+end
```

Comment:
Again – I'm not sure this class needs to exist really. It doesn't seem to do anything other than wrap `Menu`.